db/cache/charts.go

```diff
@@ -410,11 +414,12 @@ func (charts *ChartData) Lengthen() error {
 
 	// Make sure the database has set an equal number of blocks in each data set.
 	blocks := charts.Blocks
-	shortest, err := ValidateLengths(blocks.Time, blocks.PoolSize,
-		blocks.PoolValue, blocks.BlockSize, blocks.TxCount,
-		blocks.NewAtoms, blocks.Chainwork)
+	shortest, err := ValidateLengths(blocks.Height, blocks.Time,
```

Comment:
`blocks.Height` is unnecessary since the data stored is the same as the indexes of any array in the `blocks` binType.
This is how the frontend shows the height axis even without a `blocks.Height` in the backend.

Reply:
I'm gonna leave it for now, with a comment, to avoid a major version bump of the cache module.